File: backup/Search/python/01_문서검색.py
# fast campus 강의
# https://www.acmicpc.net/problem/1543
# 2

document = input()
word = input()
idx = 0
result = 0

while len(document) - idx >= len(word):
    if document[idx:idx + len(word)] == word:
        result += 1
        idx += len(word)
    else:
        idx += 1

print(result)

File: backup/Search/c++/01_문서검색.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/1543
// 2

#include <iostream>

using namespace std;

int main()
{
    string document;
    getline(cin, document);
    string word;
    getline(cin, word);

    int idx = 0, result = 0;
    while (document.length() - idx >= word.length()) {
        if (word == document.substr(idx, word.length())) {
            result++;
            idx += word.length();
        } else {
            idx++;
        }
    }
    cout << result;
}